smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/core/RuntimeTypes.kt

```diff
@@ -121,6 +121,7 @@ object RuntimeTypes {
         val urlEncodeComponent = runtimeSymbol("urlEncodeComponent", KotlinDependency.UTILS, "text")
 
         object Convenience {
```

Comment:
nit

I believe we were using nested objects to represent the package hierarchy not which file in the runtime they belong to.